controllers/users.js

```diff
@@ -563,6 +635,186 @@ class Users {
       }
     });
   }
+
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} get all users
+   */
+  static async getAllUsers(req, res) {
+    const { limit = 10, offset = 0 } = req.query;
+    const findUsers = await User.findAll({
+      attributes: [
+        'firstName',
+        'lastName',
+        'role',
+        'email',
+        'username',
+        'image'
+      ],
+      limit,
+      offset
+    });
+
+    return res.status(200).send({
+      status: 200,
+      profile: findUsers
+    });
+  }
+
+  /**
+   * This method implements the functionalities of deleting a user
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} deleted User
+   */
+  static async deleteUser(req, res) {
+    const { username } = req.params;
+    const { user } = req;
+
+    if (!user || user.role !== 'super-admin') {
```

Comment:
@shemaeric As the route is accessible to admins and super admins only(by token verification) checking this here is unnecessary.

Reply:
Same on line 732, 790